.github/workflows/unit-tests.yml

```diff
@@ -30,3 +30,5 @@ jobs:
         run: npm run unit
       - name: Run not fully integrated unit tests
         run: npm run unit-not-supported
+      - name: Run rust unit tests
+        run: cargo test
```

Comment:
:thought_balloon:  Perhaps, for better debug experience, we should enable `RUST_BACKTRACE=full`?

Reply:
I can see we have the following in our workflow yamls in Rust Driver's CI:

```yaml
env:
  CARGO_TERM_COLOR: always
  RUST_BACKTRACE: full
```